### DISCUSSION TITLE
Libre Computer Alta AI SBC with integrated NPU for Frigate

### DISCUSSION THREAD

**Discussion by dsx724** [blakeblackshear/frigate#8690] [Show and tell]
We really enjoy the Frigate project and we're investing in some upstream work for NPUs in Mesa for one of our upcoming Alta SBC that has builtin NPU for video processing. The Alta SBC as nearly the same performance as Google Coral and is priced the same as Google Coral so you get the SBC and processing capability in the same package for free.

https://hub.libre.computer/t/2023-09-25-libre-computer-aml-a311d-cc-alta-ai-sbc-announcement/2905

Running MobileNetV1: https://hub.libre.computer/t/libre-computer-cottonwood-alta-tensorflow-lite-mobilenetv1-guide/3095
Since posting that, we're already another 2.5x faster. We hope to squeeze another 2x faster by early next year.

We are working on integrating the model Frigate runs and hopefully have that by year end.

**Comment by azacock** [comment]
Any updates on this?
